extensions/content-manager/admin/src/containers/ListView/index.js

```diff
@@ -70,8 +70,29 @@ function ListView({
   const getDataRef = useRef();
   const [isLabelPickerOpen, setLabelPickerState] = useState(false);
   const [isFilterPickerOpen, setFilterPickerState] = useState(false);
+  const [relationApiData, setRelationApiData] = useState([])
   const [idToDelete, setIdToDelete] = useState(null);
   const contentTypePath = [slug, 'contentType'];
+  const relation = get(layouts, [slug, 'contentType', 'settings', 'relationName']);
+  const collectionName = get(layouts, [slug, 'contentType', 'schema', 'collectionName']);
+  const isRelationViewEnabled  = layouts[slug].contentType.settings.relation;
+
+  useEffect(() => {
+    if(isRelationViewEnabled) {
+      const getData = async () => {
+        try {
+          const relationData = await request(getRequestUrl(`${collectionName}/relation/${relation}`), {
+            method: 'GET',
+          });
+          setRelationApiData(relationData)
+
+        } catch (err) {
+          strapi.notification.error(`error occured`);
```

Comment:
Shouldn't we use translations? does strapi use translations for error notifications?

Reply:
fixed